Source/Fuse.Elements/Resources/HttpImageSource.uno

```diff
@@ -146,9 +146,9 @@ namespace Fuse.Resources
 			try
 			{
 				_loading = true;
-				if (_diskCache && IsFileCacheExist(out _filenameBase, out _contentType))
```

Comment:
This is the root cause, `IsFileCacheExist` will not be called when `_diskCache` variable is `false`. Silly me😩